lib/tok.js

```diff
@@ -362,6 +365,26 @@
       this.lastType = type;
       return type;
     },
+
+    /** Add line information to tokens. The last token
+     * on a line of code is a carriage return (⏎). Single
+     * line or multi-line comments are retrieved as a single
+     * comment token.
+     * */
+    addLineCount: function(token, type, lastValue){
+      token['line'] = this.lineCounter;
```

Comment:
Are you doing `['line']` to prevent mangling? I think it's fine to use `token.line` here.